engine/platform/src/platform_window_glfw3.cpp

```diff
@@ -142,26 +142,35 @@ namespace dmPlatform
 
     HWindow NewWindow()
     {
+        glfwSetErrorCallback(OnError);
```

Comment:
I moved the error callback setup here, since it can be done before `glfwInit`, according to the documentation.